lib/index.js

```diff
@@ -5,6 +5,20 @@ const calculateExecution = endTime =>
 
 const isEmpty = v => v === null || v === undefined;
 
+const extractEventData = (event) => {
+  if (!event) {
+    return {};
+  }
+
+  if (event.body) {
+    const body = event.isBase64Encoded ? Buffer.from(event.body, 'base64').toString() : event.body;
```

Comment:
Where does flag `event.isBase64Encoded` come from?

Reply:
https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-payload-encodings-workflow.html

Reply:
I don't see any mention of that flag there

Reply:
https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html

Reply:
```
{
    "resource": "Resource path",
    "path": "Path parameter",
    "httpMethod": "Incoming request's method name"
    "headers": {String containing incoming request headers}
    "multiValueHeaders": {List of strings containing incoming request headers}
    "queryStringParameters": {query string parameters }
    "multiValueQueryStringParameters": {List of query string parameters}
    "pathParameters":  {path parameters}
    "stageVariables": {Applicable stage variables}
    "requestContext": {Request context, including authorizer-returned key-value pairs}
    "body": "A JSON string of the request payload."
    "isBase64Encoded": "A boolean flag to indicate if the applicable request payload is Base64-encode"
}
```

Reply:
Also see any json examples in test/ dir

Reply:
Found here: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html